### PR TITLE
Sync OpenAPI spec to fix Fting issues

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

**Comment by om4james** [comment]
This looks great @logankilpatrick, thank you 👏

File: openapi.yaml [hunk 1]
```diff
@@ -1278,7 +1278,7 @@ paths:
               "hyperparameters": {
                   "n_epochs": 4,
               },
-              "trained_tokens": 5768
+              "trained_tokens": 5768 | null
```
Comment (schnerd) [inline openapi.yaml:1281]: I think these snippets should be valid json, can you remove this null?

Comment (schnerd) [inline openapi.yaml:1281]:
```suggestion
              "trained_tokens": 5768
```

**Review by schnerd — APPROVED** [review]

File: openapi.yaml (hunk 1 above)
Comment (schnerd) [reply openapi.yaml]: Updated

**Pull request merged**